app/models/changeset.rb

```diff
@@ -28,6 +28,20 @@ def commits
     @commits ||= comparison.commits.map {|data| Commit.new(repo, data) }
   end
 
+  def last_commit_status
+    if comparison.commits.nil? || comparison.commits.empty?
```

Comment:
you could use `.blank?` here http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F